src/Controllers/FilesController.js

```diff
@@ -42,22 +44,167 @@ export class FilesController extends AdaptableController {
   deleteFile(config, filename) {
     return this.adapter.deleteFile(filename);
   }
+  async canViewFile(config, fileObject, user) {
+    const acl = new Parse.ACL(fileObject.ACL);
+    if (!acl || acl.getPublicReadAccess()) {
+      return true;
+    }
+    if (!user) {
+      return false;
+    }
```

Comment:
Any way to get the standard right checker ?

Reply:
what do you mean?

Reply:
i mean that it fill me strange that File need it's own security check, since _File will be now a standard class, can we find a solution to inherit the basic class/ACL security system ?

Reply:
we have `validatePermission` performed by `schemaController` and also `addPointerPermissions` from database controller. 

May be my question here is why we need `canViewFile` since read security will be handled by Database controller and schema controller as a normal Class ? 

(I probably also missed something about your intention here 😄 )

Reply:
I have updated the code to inherit internal database ACLs :)